system/build.sh

```diff
@@ -0,0 +1,178 @@
+#!/bin/bash
+
+# Make sure we've got the compiler
+if [ "`which arm-none-eabi-gcc`" == "" ]; then
```

Comment:
Make sure `arm-none-eabi-gcc` exists and don't try building everything without it.